manifests/user.pp

```diff
@@ -18,7 +18,7 @@
 #   SuperMicro IPMI, user id 2 is reserved for the ADMIN user.
 #
 define ipmi::user (
-  String $password,
+  Variant[Sensitive[String[1]], String[1]] $password,
```

Comment:
Please add a unit test that passes a `sensitive()` value.

Reply:
Test provided... I think.